Cargo.toml

```diff
@@ -8,6 +8,9 @@ description = "Pure rust nakama protocol implementation"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[features]
```

Comment:
The futures library should only be used in tests, so normally it should be in `dev-dependencies`. However, doc tests are not using the `dev-dependencies` so a custom feature flag is used to enable `futures` for doc tests too

Reply:
Hmm, why can't the doc tests use dev-dependencies?

Reply:
Sorry I misremembered here. The problem is that I am using test helpers and only including them for tests. In doc tests the helpers are not included, thus this workaround, see https://github.com/rust-lang/rust/issues/45599